apisix/cli/ops.lua

```diff
@@ -713,8 +713,16 @@ local function start(env, ...)
         util.die("Error: It is forbidden to run APISIX in the /root directory.\n")
     end
 
-    local cmd_logs = "mkdir -p " .. env.apisix_home .. "/logs"
-    util.execute_cmd(cmd_logs)
+    local logs = env.apisix_home .. "/logs"
+    local logs_path = pl_path.exists(logs)
+    if not logs_path then
```

Comment:
We can use `if not pl_path.exists`? The `exists` will return the input argument back if it exists, so there is no need to reassign it back.